TheEthicalHackerCup/Assets/Scenes/AttackMenu.unity

```diff
@@ -217,379 +464,3230 @@ Transform:
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 491291825}
-  m_Material: {fileID: 0}
-  m_IsTrigger: 0
+  m_GameObject: {fileID: 119429745}
+  m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
+  m_LocalPosition: {x: 0, y: 0, z: 0}
+  m_LocalScale: {x: 1, y: 1, z: 1}
+  m_ConstrainProportionsScale: 0
+  m_Children:
+  - {fileID: 195546004}
+  - {fileID: 1619254608}
+  m_Father: {fileID: 835391195}
+  m_RootOrder: 4
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: 0, y: 0}
+  m_SizeDelta: {x: 100, y: 100}
+  m_Pivot: {x: 0.5, y: 0.5}
+--- !u!4 &146551280 stripped
+Transform:
+  m_CorrespondingSourceObject: {fileID: 914061203414692282, guid: 67782a0a5b8eb7b4f9810af895a44bff, type: 3}
+  m_PrefabInstance: {fileID: 1737642615}
+  m_PrefabAsset: {fileID: 0}
+--- !u!1 &195546003 stripped
+GameObject:
+  m_CorrespondingSourceObject: {fileID: 1583904904307026, guid: 9f09a67aeaf80e349b443646575a4e8a, type: 3}
+  m_PrefabInstance: {fileID: 763943152}
+  m_PrefabAsset: {fileID: 0}
+--- !u!224 &195546004 stripped
+RectTransform:
+  m_CorrespondingSourceObject: {fileID: 224879373836950858, guid: 9f09a67aeaf80e349b443646575a4e8a, type: 3}
+  m_PrefabInstance: {fileID: 763943152}
+  m_PrefabAsset: {fileID: 0}
+--- !u!114 &195546005
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 195546003}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: e0f25e5357f4c1a47a6346c440414d02, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+--- !u!1 &201727700 stripped
+GameObject:
+  m_CorrespondingSourceObject: {fileID: 1583904904307026, guid: 9f09a67aeaf80e349b443646575a4e8a, type: 3}
+  m_PrefabInstance: {fileID: 205537921}
+  m_PrefabAsset: {fileID: 0}
+--- !u!224 &201727701 stripped
+RectTransform:
+  m_CorrespondingSourceObject: {fileID: 224879373836950858, guid: 9f09a67aeaf80e349b443646575a4e8a, type: 3}
+  m_PrefabInstance: {fileID: 205537921}
+  m_PrefabAsset: {fileID: 0}
+--- !u!114 &201727704
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 201727700}
   m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: e0f25e5357f4c1a47a6346c440414d02, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+--- !u!1001 &205537921
+PrefabInstance:
+  m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Size: {x: 1, y: 1, z: 1}
-  m_Center: {x: 0, y: 0, z: 0}
---- !u!23 &491291827
-MeshRenderer:
+  m_Modification:
+    m_TransformParent: {fileID: 1095828411}
+    m_Modifications:
+    - target: {fileID: 1583904904307026, guid: 9f09a67aeaf80e349b443646575a4e8a, type: 3}
+      propertyPath: m_Name
+      value: ManInMIddleBar
```

Comment:
Man in the Middle doesn't exist anymore. This appears in the AttackMenu Scene hierarchy under phishing and gives an error in Unity editor